packages/hardhat-zksync-solc/README.md

```diff
@@ -43,22 +44,29 @@ zksolc: {
     }
 },
 ```
+
+Usage of zksolc compiler version greater or equal to 1.5.0
+
+- It's necessary to set the viaYul or viaEVMAssembly flag to true at zksolc settings to specify the compiler codegen manually.
```

Comment:
It is mandatory to specify the compiler codegen manually. This can be done by setting either the viaYul or viaEVMAssembly flag to true in the zksolc settings. Only one of these flags can be set to true at a time, ensuring that the code generation process is clearly defined.